category/category_test.go

```diff
@@ -0,0 +1,47 @@
+package category_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/almighty/almighty-core/category"
+	"github.com/almighty/almighty-core/convert"
+	"github.com/almighty/almighty-core/gormsupport"
+	"github.com/almighty/almighty-core/resource"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+var plannerTestIssues1 = uuid.FromStringOrNil("5cccf59d-fed6-4c4a-a420-eebe83fe09e1")
+var plannerTestIssues2 = uuid.FromStringOrNil("5cccf59d-fed6-4c4a-a420-eebe83fe09e1")
+
+const plannerTestIssuesName = "planner.testIssues"
+
+func TestCategory_Equal(t *testing.T) {
```

Comment:
I didn't see a test that `a.Equal(b)` actually returned `true` when `a` and `b` are equal.